index.html

```diff
@@ -44,6 +46,219 @@ <h3>I'am a Software Developer</h3>
+          industry's standard
+        </p>
+        
+        <ul class="buttons1">
+          <li><button class="button1">html</button></li>
+          <li><button class="button1">bootstrap</button></li>
+          <li><button class="button1">Ruby</button></li>
+        </ul>
+        <br>
+        <div>
+          <button class="orange-button orange-button2">See Project</button>
+        </div>
+      </section>
+
+      <section class="card2">
+        <h3>Professional Art <br> Printing Data</h3>
+      
+        <p>A daily selection of privately 
+          personalized reads; no accounts or 
+          sign-ups required. has been the 
+          industry's standard
+        </p>
+        
+        <ul class="buttons1">
+          <li><button class="button1">html</button></li>
+          <li><button class="button1">bootstrap</button></li>
+          <li><button class="button1">Ruby</button></li>
+        </ul>
+        <br>
+        <div>
+          <button class="orange-button orange-button2">See Project</button>
+        </div>
+      </section>
+
+      <section class="card2">
+        <h3>Professional Art <br> Printing Data</h3>
+      
+        <p>A daily selection of privately 
+          personalized reads; no accounts or 
+          sign-ups required. has been the 
+          industry's standard
+        </p>
+        
+        <ul class="buttons1">
+          <li><button class="button1">html</button></li>
+          <li><button class="button1">bootstrap</button></li>
+          <li><button class="button1">Ruby</button></li>
+        </ul>
+        <br>
+        <div>
+          <button class="orange-button orange-button2">See Project</button>
+        </div>
+      </section>
+
+      <section class="card2">
+        <h3>Professional Art <br> Printing Data</h3>
+      
+        <p>A daily selection of privately 
+          personalized reads; no accounts or 
+          sign-ups required. has been the 
+          industry's standard
+        </p>
+        
+        <ul class="buttons1">
+          <li><button class="button1">html</button></li>
+          <li><button class="button1">bootstrap</button></li>
+          <li><button class="button1">Ruby</button></li>
+        </ul>
+        <br>
+        <div>
+          <button class="orange-button orange-button2">See Project</button>
+        </div>
+      </section>
+    </section>
+
+    <section class="container2">
+      <section class="aboutMe">
+        <h1>About me</h1>
+
+        <p>
+          Hello I’m a software developer! I can help 
+          you build a product , feature or website 
+          Look through some of my work and 
+          experience! If you like what you see and 
+          have a project you need coded, don’t 
+          hestiate to contact me.
+        </p>
+
+        <div>
+          <button class="orange-button orange-button3">Get My Resume</button>
+        </div>
+      </section>
+
+      <hr class="line">
+
+      <section class="categories">

```

Comment:
- [ ] The about me section should be centred just as it is in the figma design.

Yours | Original
----- | -----
![image](https://user-images.githubusercontent.com/16058470/121603678-0c1f6080-ca41-11eb-87e1-5d53152a2823.png) | ![image](https://user-images.githubusercontent.com/16058470/121603733-23f6e480-ca41-11eb-99e8-e6a08f3ea783.png)

Reply:
Hey @chaw-bot , you've fixed this issue!